src/uucore/src/lib/features/format/num_parser.rs

```diff
@@ -181,11 +181,13 @@ impl ParsedNumber {
             };
         }
 
+        let trimmed_input = input.trim_ascii_start();
+
         // Initial minus sign
-        let (negative, unsigned) = if let Some(input) = input.strip_prefix('-') {
```

Comment:
Not a big deal, but the lack of symmetry is not so nice, maybe.

I wonder if you want to do this intead:
```
let (negative, unsigned) = if let Some(trimmed_input) = trimmed_input.strip_prefix('-') {
            (true, trimmed_input)
} else {
            (false, trimmed_input)
}
```

Or just shadow input above:
```
let input = input.trim_ascii_start();
```